test/util/login-helpers.js

```diff
@@ -52,16 +51,29 @@ function initUserMock(level) {
 }
 /* eslint-enable camelcase */
 
-function logIn(client, url, level) {
-  initUserMock(level);
+let tokenCounter = 0;
+
+function getUserToken() {
+  return `token-${process.pid}-${tokenCounter++}`;
+}
+
+function logIn(token, client, url, level) {
+  initUserMock(token, level);
+
   client
     .url(`${E2eHelpers.baseUrl}${url}`)
     .waitForElementVisible('body', Timeouts.normal);
```

Comment:
Is this because localStorage isn't set until the body is already loaded?  If that's the case, I'd guess that the URL paramater approach would be the way to go.  It would just take a couple of changes in `test-server.js`, I think. 

Something for a future PR, perhaps!